components/Pages/Web3Page/Trailer.tsx

```diff
@@ -13,7 +20,17 @@ const Trailer = ({ isPopup = false }) => {
       <div className="flex flex-col items-center gap-y-[15px] md:gap-y-[20px]">
         <div className="flex flex-col gap-y-[10px]">
           <ZoraDropPlayers isPopup={isPopup} />
-          <CollectAllButton />
+          {/* eslint-disable-next-line no-nested-ternary */}
+          {authenticated ? (
+            isLoggedByEmail ? (
+              <PrivyCollectButton />
+            ) : (
+              <CollectAllButton />
+            )
+          ) : (
+            <LoginButton />
+          )}
```

Comment:
please do not nest this ternary

right now, you have `3 buttons` and are using a `nested ternary` to render them conditionally.

I suggest: `1 button` with an `onClick` handler. The `function` contains `if` statements from this nested ternary. 
- Cleaner code. 
- Less duplicated `button` code.
- no blocking of `linter` warnings.